### PR TITLE
feat(explore): add inspect action

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,10 @@
         "command": "pw.extension.install",
         "title": "Install Playwright"
       },
+      {
+        "command": "pw.extension.command.inspect",
+        "title": "Inspect"
```
Comment (dgozman) [inline package.json:48]: Perhaps something more descriptive like `Pick selector`?

File: src/reusedBrowser.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    const node = await this._playwrightTest.findNode();
+    const allArgs = [
+      config.cli,
+      'run-server',
+      '--reuse-browser',
+      `--path=/${createGuid()}`
+    ];
+    this._serverProcess = spawn(node, allArgs, {
+      cwd: config.workspaceFolder,
+      stdio: ['pipe', 'pipe', 'pipe', 'ipc'],
```
Comment (dgozman) [inline src/reusedBrowser.ts:66]: When piping stdout/stderr, we need to drain them, or it could stall on Windows.

File: src/reusedBrowser.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+      env: { ...process.env, PW_CODEGEN_NO_INSPECTOR: '1' },
+    });
+
+    this._serverProcess.on('exit', () => {
+      this._setBrowserServerWS(undefined);
+      this._serverProcess = undefined;
```
Comment (dgozman) [inline src/reusedBrowser.ts:72]: We should resolve/reject the promise from line 79 here to avoid stalling.

File: src/reusedBrowser.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    this._send('inspect', { enabled: true });
+    this._selectorExplorerBox = this._vscode.window.createInputBox();
+    this._selectorExplorerBox.title = 'Selector explorer';
+    this._selectorExplorerBox.value = '';
+    this._selectorExplorerBox.ignoreFocusOut = true;
+    this._selectorExplorerBox.onDidChangeValue(selector => {
+      this._send('highlight', { selector });
+    });
+    this._selectorExplorerBox.onDidHide(() => {
+      this._send('inspect', { enabled: false });
```
Comment (dgozman) [inline src/reusedBrowser.ts:119]: `this._selectorExplorerBox = undefined;`

**Review by dgozman — APPROVED** [review]

File: src/reusedBrowser.ts (hunk 3 above)
Comment (pavelfeldman) [reply src/reusedBrowser.ts:72]: Sure

**Pull request merged**